test/Ocelot.AcceptanceTests/Authentication/AuthenticationSteps.cs

```diff
@@ -101,9 +101,9 @@ public void WithJwtBearerAuthentication(IServiceCollection services, bool addOce
         app.MapGet("/connect", () => "Hello! Connected!");
         app.MapPost("/token", (AuthenticationTokenRequest model) =>
         {
-            if (!apiScopes.Contains(model.Scope))
+            if (!apiScopes.Intersect(model.Scopes.Split(' ')).Any())
```

Comment:
The code looks fine.
Since this isn't part of the core library, heavy use of LINQ to Objects isn't a concern.
FYI, I might change this logic in future PRs, but the splitting will remain.